app/config.py

```diff
@@ -180,8 +187,11 @@ def __getattr__(self, name):
     def __getitem__(self, name):
         """Gets a configuration setting for this repository.  Looks for a
         repository-specific setting, then falls back to a global setting."""
-        return get_for_repo(self.repo, name)
```

Comment:
I couldn't find any get_for_repo() calls from within a handler (i.e., where a Configuration object is available).

Reply:
Thanks! Maybe can you add a comment in config.get_for_repo() and config.get() to warn that they are not as efficient as instance methods of Config class?

It may also be better to update https://github.com/google/personfinder/wiki/Config to suggest using Config object in the handlers. The doc currently only mentions config.get() and config.get_for_repo().